### PR TITLE
add omnicored 0.3.1

### PR DESCRIPTION
**Pull request by dancju**
Would you consider adding version 0.3.1 as there are some major changes such as segwit support?

### CODE REVIEW

File: v0.3.1-alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
+FROM alpine:3.8
+
+LABEL maintainer="Maksym Pugach <pugach.m@gmail.com>"
```
Comment (mpugach) [inline v0.3.1-alpine/Dockerfile]: please add yourself to the maintainers list

File: v0.3.1-alpine/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    rm -rf $BDB_PREFIX/docs && \
+    rm -rf $BITCOIN_REPO && \
+    strip $BITCOIN_ROOT/bin/omnicore-cli && \
+    strip $BITCOIN_ROOT/bin/bitcoin-tx && \
+    strip $BITCOIN_ROOT/bin/omnicored && \
+    strip $BITCOIN_ROOT/lib/libbitcoinconsensus.a && \
+    strip $BITCOIN_ROOT/lib/libbitcoinconsensus.so.0.0.0 && \
+    apk del git autoconf automake g++ make file
+
+COPY docker-entrypoint.sh /entrypoint.sh
```
Comment (mpugach) [inline v0.3.1-alpine/Dockerfile:50]: and place the file into the folder

**Review by mpugach — CHANGES_REQUESTED** [review]
thank you @nerdDan, please check the comments

**Review by mpugach — APPROVED** [review]
👍

**Comment by dancju** [comment]
Would you mind publish it to `hub.docker.com`?

**Comment by mpugach** [comment]
building

**Pull request merged**